appendix.Rmd

```diff
@@ -2,43 +2,65 @@
 
 # Post template in Markdown {#templatemd}
 
-INSTRUCT author to copy this template, or get it from roblog
+[Markdown template](/templates/post-template.md) to be saved as `/content/blog/YYYY-MM-DD-slug/index.md`
```

Comment:
This link doesn't work in the gitbook, I'll remove it since the template is just below the lines.